ledger/acctdeltas_test.go

```diff
@@ -50,22 +50,22 @@ import (
 	"github.com/algorand/go-algorand/util/db"
 )
 
-func checkAccounts(t *testing.T, tx *sql.Tx, rnd basics.Round, accts map[basics.Address]basics.AccountData) {
-	arw := store.NewAccountsSQLReaderWriter(tx)
+func checkAccounts(t *testing.T, tx store.TransactionScope, rnd basics.Round, accts map[basics.Address]basics.AccountData) {
```

Comment:
we need to stick to a common name for `store.TransactionScope`, now it is either ts or tx